app/src/main/java/com/hedvig/app/feature/profile/ui/charity/CharityAdapter.kt

```diff
@@ -40,18 +40,18 @@ class CharityAdapter(
         val title: TextView = binding.cashbackOptionTitle
         val paragraph: TextView = binding.cashbackOptionParagraph
         val button: Button = binding.cashbackSelect
-        fun bind(item: ProfileQuery.CashbackOption, clickListener: (String) -> Unit) {
+
+        fun bind(charityOption: CharityOption, clickListener: (id: String) -> Unit) {
             binding.apply {
-                cashbackOptionTitle.text = item.name
-                cashbackOptionParagraph.text = item.paragraph
+                cashbackOptionTitle.text = charityOption.name
+                cashbackOptionParagraph.text = charityOption.description
```

Comment:
This change was due to the type `Cashback` having a deprecated field.
`paragraph: String @deprecated(reason: "use description instead")`

Reply:
Did you verify the correctness of this change? 😬 Sometimes these deprecations are not actually proper replacements, and hence they have to be vetted for every use-case.

Reply:
Oh, okay I will double check this then, I didn't know we'd deprecate something with the message "use X instead" and not actually have it be a replacement :D
From running the app and looking at differences it looked fine, but I will admit I wasn't expecting this really to be a problem at all so I didn't pay 100% attention. I shall check again 😅

Reply:
Yeah just checked in the graphql playground with all locales for all markets and they all respond with the exact same description and paragraph field.